pulp_2to3_migration/app/migrations/0001_initial.py

```diff
@@ -12,7 +12,7 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('core', '0018_auto_20191127_2350'),
+        ('core', '0019_add_signing_service_model'),
```

Comment:
This migration isn't in 3.0.1, only 3.1 :(  so this can't be installed with 3.0.1